tests/rust/log_benchmark/src/image.rs

```diff
@@ -29,8 +25,7 @@ fn prepare() -> Vec<u8> {
     //         i as u8
     //     },
     // );
-
-    image
+    // image
```

Comment:
we should still return it though

Reply:
we do, there's no `;` further up. Otherwise this wouldn't compile :)
Had to comment out the returning line because clippy complained otherwise